lib/prop_schema/generator/base_properties.ex

```diff
@@ -71,6 +72,30 @@ defmodule PropSchema.BaseProperties do
     end
   end
 
+  def generate_prop(field, :binary_id, %{required: true}) do
+    quote do
+      {unquote(Atom.to_string(field)), unquote(uuid_generator())}
+    end
+  end
+
+  def generate_prop(field, :binary_id, %{required: false}) do
+    quote do
+      {unquote(Atom.to_string(field)), StreamData.one_of([unquote(uuid_generator()), StreamData.constant(nil)])}
+    end
+  end
+
+  def generate_prop(field, :utc_datetime, %{required: true}) do
```

Comment:
Can we add a `now: true` match on this so that people can come through and add a not current_datetime version later?